CMakeLists.txt

```diff
@@ -151,6 +151,10 @@ if(MSVC)
   set(CMAKE_MSVCIDE_RUN_PATH "\$(SolutionDir)/3rd-party/ltdl/\$(Configuration);\$(SolutionDir)/src/\$(Configuration);\$(SolutionDir)/src/mc_control/fsm/\$(Configuration)")
 endif(MSVC)
 
+if(WIN32)
```

Comment:
should be `if(MSVC)` (and thus in the if above)

Reply:
I moved to `install_mc_rtc_lib` and corrected (0e665f5)